### ISSUE TITLE
[filter] Order of setting the properties

### ISSUE THREAD

**Issue by kparichay** [nnstreamer/nnstreamer#2650]
With the introduction of two versions of GstTensorFilterFramework, there has come up a bug with setting the properties.
The order of properties has now become important as setting certain properties depends on the version of the framework.
So, the version of framework is needed to set those properties, which creates a required order among some of the properties.

Given the two pipelines below:
```
pipelineA = "tensor_filter accelerator=true:GPU framework=tensorflow_lite"
pipelineB = "tensor_filter framework=tensorflow_lite accelerator=true:GPU"
```
Both the pipelines would have the correct framework. However, `accelerator` for `pipelineA` would be set to NULL because when `accelerator` property is set, `framework` is not yet set. This issue exists for properties `accelerator` and `is_updatable`. 

This is the inherent issue behind issue #2465

**Comment by myungjoo** [comment]
This is a critical issue that is required to be resolved before the RC1 release of Tizen 6.0 M2.

**Comment by myungjoo** [comment]
Nice catch. Please fix it so that the order of properties won't affect the behavior.

**Comment by kparichay** [comment]
@myungjoo Bug fix has been proposed with #2651 but it exposes some issues with unittests on nnfw on aarch64 and tflite edge tpu on x86_64. Detailed check on these unit tests will be done soon.
